src/components/Modals/EditAssetPackModal/EditAssetPackModal.tsx

```diff
@@ -42,7 +43,16 @@ export default class EditAssetPackModal extends React.PureComponent<Props, State
       title: assetPack.title,
       thumbnail: assetPack.thumbnail,
       userId: assetPack.userId,
-      assets: assetPack.assets.map(asset => ({ ...asset, contents: {} } as RawAsset))
+      assets: assetPack.assets.map(asset => {
+        return {
+          ...asset,
+          url: basename(asset.url),
+          contents: Object.keys(asset.contents).reduce<Record<string, Blob>>((acc, path) => {
+            acc[path] = ('https://builder-api.decentraland.zone/v1/storage/assets/' + asset.contents[path]) as any
```

Comment:
☝️ `${BUILDER_SERVER_URL}/storage/assets/${asset.contents[path]}`

Also, should we make a helper for this? If you search for `${BUILDER_SERVER_URL}/storage` there are 13 results in 6 files without counting this.

Reply:
Oops didn't read this, it was a draft pr and was just quickly testing stuff, please check now!